### DISCUSSION TITLE
Call for getListing is not returning data for the sku-array anymore

### DISCUSSION THREAD

**Discussion by mkuenzli** [etsy/open-api#942] [Q&A] [answered]
Hello,

we are currently facing an issue during the getListing-Call, where the array for 'skus' is not returning any data. Its always empty, no matter if you do the call for a single variation or multi variation listing.

    "skus": [],

https://developer.etsy.com/documentation/reference/#operation/getListing
GET /v3/application/listings/{listing_id}

We need these information on a daily basis to reproduce or comprehend individual cases on customer side.
Can you check why these information are not given in the response anymore?

Thanks in advance.

Best regards
Maurice

**Comment by Gareth064** [comment]
I have just checked this for one of my listings and SKU's are being returned

![image](https://user-images.githubusercontent.com/22307223/230692289-040ae58a-2f0f-48f0-8e60-62d66e0bad5d.png)

**Reply by showtroylove** [reply, marked as answer]
@mkuenzli: This is likely because you're calling it from a non-shop owner account or have not supplied an OAuth header in your request:

Reading the API endpoint documentation in question:

> A list of SKU strings for the listing. SKUs will **only appear if the requesting user owns the shop and a valid matching OAuth 2 token** is provided. When requested without the token it will be an empty array.

This method maybe used generally to get any listing.  If you're the shop owner it will provide additional details, however.  The documentation could more clearly state this when describing the permission.  I believe you need scope listing_r, at a minimum.

**Reply by mkuenzli** [reply]
Hi together,

thanks for your replies.
Well - think i just did not read the documentation here properly. In the past the call for v3 was working like the old one for v2, where this was definitely **not** necessary. Maybe this changed during the shutdown of v2.
Will adjust this, thank you very much!
